Makefile

```diff
@@ -96,8 +96,8 @@ ifeq (debug,$(findstring debug,$(COSMOS_BUILD_OPTIONS)))
   BUILD_FLAGS += -gcflags "all=-N -l"
 endif
 
-#? all: Run tools build lint test vulncheck
-all: tools build lint test vulncheck
+#? all: Run tools build 
+all: build
```

Comment:
Why this change? That doesn't look like codeql related

Reply:
Codeql attempts to run a few build commands one of which is make. So minting and all was running here on top of the other action